rohan-calculator/script.js

```diff
@@ -0,0 +1 @@
+console.log("Hello world!")
```

Comment:
We don't need this console.log so you can remove it.

Reply:
okay i will do it.